packages/app/src/pages/WalletPage.tsx

```diff
@@ -1,79 +0,0 @@
-import toast from "react-hot-toast";
```

Comment:
Also don't remove this screen maybe we can improve it on a next issue

Reply:
🤔 but this page doesn't need to exist anymore, it becomes the Assets page. We don't need a wallet page like this one, because now we have an assets page in the menu and Wallet feature on topbar.